WDL/runtime/task.py

```diff
@@ -248,13 +251,13 @@ def _run(self, logger: logging.Logger, command: str) -> int:
                 logger.debug(
                     "docker container name = {}, id = {}".format(container.name, container.id)
                 )
-
                 # long-poll for container exit
                 while exit_info is None:
                     try:
+                        for line in pygtail:
+                            logger.info(f"StdError: {line}")
```

Comment:
we can log `line.rstrip()` here and below, since logger itself adds a newline to the message you give it